src/views/computational-simulation/StructureDesignView.vue

```diff
@@ -0,0 +1,389 @@
+                                <template #caption>
+                                    Figure 5: The effect of insertions and deletions on the bending of a structure, shown from a geometric perspective in CanDO 
+                                </template>
+                            </CaptionedGraphics>
+                        </li>
+
+                        <li>
+                            <b>Uploading to Nanoval and Computational Analysis:</b> The structure was uploaded to the nanoval library for manual inspection of sandwich or staple strands. After correction, we then obtained Root Mean Square Fluctuation Values and Stability Maps using CanDO, which helped in computationally predicting the structural stability and topology. Ensuring the simulation displayed the intended shape and stability was key to averting the formation of kinetic traps and sandwich strands. 
+                        </li>
+
+                        <li>
+                            <b>Cross-Comparisons with Existing Literature:</b> Once we acquired Perrault et al.'s structure (Rajkumar, Perrault, UBC BIOMOD, personal correspondence), we undertook a comparative analysis between our design and existing structures documented in the literature.
+                        </li>
+                    </ol>
+                </p>
+            </template>
+        </SingleColumn>
+
+        <SingleColumn>
+            <template #title> Results and Discussion </template>
+            <template #body>
+                <p>
+                    We primarily used CanDO for structure prediction. This tool  is useful if we effectively eliminate the requirement for mechanism-dependent fluctuations — variations in the DNA structure that could be influenced by adjacent DNA strands, such as the nanohinge done by our team previously (UBC BIOMOD 2022). In CanDO, the Root Mean Square Deviation (RMSD) is visualized through a gradient transitioning from red to blue, illustrating the average distance molecules stray from their starting positions. Importantly, our post-design evaluation revealed no major discrepancies, and the peak Root Mean Square Fluctuation (RMSF) values remained notably low for all studied structures. The subsequent schematic illustrates the minimum, maximum, and average RMSF values, all presented within the 95% CI, for the 3-gon, 4-gon, and 5-gon, respectively. These are paired with their respective CanDO images. To clarify, when we reference the "RMSF values at the 95% CI," we are indicating the range of RMSF values (minimum, maximum, and average) for each structure, where we possess 95% confidence that the actual RMSF value resides within this interval. A broad confidence interval signifies increased uncertainty, while a tighter interval reflects a higher level of precision.
+                </p>
+
+                <section class="py-2" v-for="(structure, i) in structures">
+                    <h3 class="text-base text-white"> {{ structure.label }} </h3>
```

Comment:
I would use sectionStyle for the styling here just for consistency

Reply:
Gotcha gotcha